Readme File (#82)
FACG6/AMiDEAST: README.md

# AMiDEAST

> AMIDEAST is an website that allows the staff members add and view courses,it can also mange students,in the other hands the student can see the proposed courses and select one to apply for.

Reviewer: ```suggestion
AMIDEAST is a website that allows the staff members to add and view courses, it can also manage students, in the other hands the student can see the proposed courses and select one to apply for.
```

## Problem statement

> - English Language (EL) staff spend alot of time collecting registration, and doing refund and texting students to get thier money back.

Reviewer: ```suggestion
- English Language (EL) staff spend a lot of time in collecting registrations, and doing refund and texting students to get thier money back.
```

- difficulty in reaching the students who are already enrolled in AMIDEAST.

## Our Solution

- bulid website allows the staff to add and view courses so that we reduce time and effort.

> - manger can determine students prefered time/days.

Reviewer: ```suggestion
- manager can determine students preferred time/days.
```

## User Stories

#### As a Member of EL's staff :

- I want to login to my account. so that my account will be protected.
- I want to add a new course. so that the student can apply to the courses.
- I want to view the courses that I published it before. so that I can select the suitable dates.
- I want to search for a course. so that I can find any course easily.
- I want to add a new student. so that they can apply to the courses.
- I want to edit any student. so that I can restore any data that got wrong or forgotten.
- I want to view a student so that I can access data and edit it.

> - I want to search for a student so that I can find any student easily and without

Reviewer: ```suggestion
- I want to search for a student so that I can find any student easily and without
```
without what ?

---

#### As a Student :

- I want to login to my account, so that my account will be protected.
- I want to see the courses so that I can apply to it.
- I want to apply to a course so that I want to take it.
- I want to see my information so that I can check my info.

> ### Devloper Experince:

Reviewer: ```suggestion
### Developer Experience :
```

1- clone our repo.

`git clone https://github.com/FACG6/AMiDEAST.git`

2- `cd AMiDEAST`

> 3- `npm i`

Reviewer: ```suggestion
3- ```  cd Client && npm i && cd ../server && npm i && ..  ```
```
